src/components/profile/ProfileHeader.jsx

```diff
@@ -78,7 +84,8 @@ class ProfileHeader extends Component {
   );
 
   render() {
-    const { username, remote } = this.state;
+    const { username, remote, modalShow } = this.state;
+    const handleClose = () => this.setState({ modalShow: false });
```

Comment:
Can we just make this a separate function like `renderModal` instead of having it hidden in the render function.  Otherwise, everything else looks good!!